test/unit/test_gcs_util.py

```diff
@@ -22,6 +20,13 @@
 except ImportError:
     SnowflakeGCSUtil = None
 
+try:
+    from snowflake.connector.vendored import requests  # NOQA
+    vendored_request = True
+except ImportError:  # pragma: no cover
+    import requests
+    vendored_request = False
+
```

Comment:
Same here. I do not see a good reason to keep the try/except :)

Reply:
We need these for our `OldDriver` tests. We run most up to date tests with the oldest supported driver version

Reply:
OK.Fair enough. Happy to add this comment instead of removing it.